cmd/glusterblock/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	gfd "github.com/gluster/gluster-csi-driver/pkg/glusterblock"
+	"github.com/gluster/gluster-csi-driver/pkg/glusterblock/config"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	// #nosec
+	_ = flag.Set("logtostderr", "true")
+}
+
+func main() {
+	// #nosec
+	_ = flag.CommandLine.Parse([]string{})
+	var config = config.NewConfig()
+
+	cmd := &cobra.Command{
+		Use:   "glusterblock-csi-driver",
+		Short: "GlusterFS Block CSI driver",
```

Comment:
`GlusterFS Block CSI driver`  or `Gluster loop back CSI driver`?

Reply:
glusterblock-csi-driver to glusteloopback-csi-driver ?

Reply:
again, as said, lets avoid 'loop', and fine to call 'virtual'